.travis.yml

```diff
@@ -28,13 +26,13 @@ install:
 - pip install git+https://github.com/nerdoftech/s3_index_generator.git
 script:
 - css-validator f5_sphinx_theme/static/css/custom.css
-- htmlhint f5_sphinx_theme
+# - htmlhint f5_sphinx_theme
 - export PATH="$PATH:$(pwd)/node_modules/.bin"
 - webpack
-- "./test/test-docs-docker.sh"
+- cd test && make html
 
 before_deploy:
-  - echo "Deploying to ${S3_DIST_URL}/${UPLOAD_DIR}/index.html"
+  - echo "Deploying to ${S3_DIST_URL}/${UPLOAD_DIR}/"
```

Comment:
if we're removing the "index.html", we can take out the trailing slash as well.

Reply:
IIRC correctly, not having the trailing slash gives you an access denied error.

Reply:
This is just an echo statement, it doesn't impact the deployment.